documentation/peer-testing-1/heuristic-evaluation-experiment-instructions.md

```diff
@@ -0,0 +1,27 @@
+# Heuristic evaluation experiment instructions #
+- Heuristic evaluation is a thorough assessment of a product's user interface, and its purpose is to detect usability issues that may occur when users interact with a product, and identify ways to resolve them.
+
+## 10 User Interface Design Heuristics ##
+1. **Visibility of system status.** system status should be displayed when operations are executed like a toast message. For example, When a button is onclicked, a toast message or any other forms of message should tell user what button they are clicking on.
+2. **Match between system and the real world.** System should be designed using such as icons, languages, syntax from real world. For example, Icons like calendars should be a real mini-calendar image to indicate that by clicking this icon a calandar will be displayed.
+3. **User control and freedom.** System are avaliable to be undoable or redoable. For example, 'back' buttons and 'undo' buttons.
+4. **Consistency and standards.** UI designers should make sure that staffs such as frames, icons, and font under the similar or even the same categories should be consistent.
+5. **Error prevention.** Technical problems should not occur during User operations. Eliminating or flagging actions that may result in errors are two possible means of achieving error prevention.
+6. **Recognition rather than recall.** UI designers should minimize the use of user's memory when they are guided to use the system. Recognizing something is always easier than recall because recognition involves perceiving cues that help us reach into our vast memory. For example, in some shopping website, there is a recently-viewed list to help users remind what they are interested in.
```

Comment:
The second sentence has been copied and pasted directly from https://www.interaction-design.org/literature/article/heuristic-evaluation-how-to-conduct-a-heuristic-evaluation?ref=seoptimer.com